src/index.ts

```diff
@@ -34,9 +34,13 @@ app.use('/admin/graphql', graphqlHTTP({
 }));
 
 app.get('/healthz', async (_, response) =>{
-  await checkDB();
-
-  response.status(200).send('OK');
+   try {
+     await checkDB();
+     response.status(200).send('OK');
+   } catch (e: Error | any) {
```

Comment:
Siempre que claves un `any` en cualquier lado básicamente estás skippeando cualquier sistema de tipos.

Acá sería `Error | unknown`

Reply:
https://dev.to/tomdohnal/typescript-any-vs-unknown-a-deep-dive-3iem